userspace/libsinsp/sinsp.cpp

```diff
@@ -1661,7 +1661,7 @@ void sinsp::start_dropping_mode(uint32_t sampling_ratio)
 }
 #endif // _WIN32
 
-void sinsp::set_filter(std::unique_ptr<sinsp_filter> filter)
+void sinsp::set_filter(std::unique_ptr<sinsp_filter> filter, const std::optional<std::string>& filterstring)
```

Comment:
```suggestion
void sinsp::set_filter(std::unique_ptr<sinsp_filter> filter, const std::string& filterstring = "")
```
I think the previous suggestion was referring to something like this -- otherwise it'd become a breaking change for whoever uses this method.